raiden/tests/unit/transfer/test_node.py

```diff
@@ -54,3 +72,81 @@ def test_is_transaction_effect_satisfied(chain_state, token_network_id, netting_
     iteration = state_transition(chain_state=chain_state, state_change=channel_settled)
 
     assert is_transaction_effect_satisfied(iteration.new_state, transaction, state_change)
+
+
+def test_get_network(chain_state, token_network_id):
+    token_address = factories.make_address()
+    token_network = TokenNetworkState(address=token_network_id, token_address=token_address)
+    payment_network = PaymentNetworkState(
+        address=factories.make_address(), token_network_list=[token_network]
+    )
+    chain_state.identifiers_to_paymentnetworks[payment_network.address] = payment_network
+    assert get_networks(
```

Comment:
`get_networks` looks like a view to me, shouldn't it be moved to the `views` module?

Reply:
done